speech_assets/SampleUtterances.txt

```diff
@@ -1,18 +1,36 @@
-AMAZON.StopIntent cancel
-AMAZON.StopIntent shut up
-AMAZON.StopIntent stop
 AddonExecute execute addon {Addon}
 AddonExecute execute plugin {Addon}
 AddonExecute execute script {Addon}
+AddonExecute execute the addon {Addon}
+AddonExecute execute the plugin {Addon}
+AddonExecute execute the script {Addon}
+AddonExecute execute the {Addon}
```

Comment:
It is advised to use them if they are valid phrase people might use.

> **Sample Utterance Contents**

> Given the flexibility and variation of spoken language in the real world, there will often be many different ways to express the same request. For example, to ask for a horoscope a user might say:
> * what is the horoscope
> * get me my horoscope
> * tell me the horoscope
> * how’s my horoscope today

> Or any other variations on the above forms:
> * “what’s” and “what is”
> * “get”, “tell”, and “give”
> * “my” and “the”

https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/defining-the-voice-interface#h2_sample_utterances

The lack of them does cause problems.

### Current sample utterance
`WatchMovie play movie {Movie}`

![screen_shot_macbook_pro_2017-01-10_at_18 48 57](https://cloud.githubusercontent.com/assets/5680968/21821398/5989d6c0-d76b-11e6-9e70-088cb3b08a57.png)

(followed by: (unrelated but might be useful))

![screen_shot_macbook_pro_2017-01-10_at_18 50 35-1](https://cloud.githubusercontent.com/assets/5680968/21821500/ba751d28-d76b-11e6-978b-a84febbbf1fe.png)